### PR TITLE
RCT-283: React Router Basics

### PR DESCRIPTION
**Pull request by ct-aa**
# Related Tickets
[RCT-283](https://codeandtheory.atlassian.net/browse/RCT-283)
[RCT-279](https://codeandtheory.atlassian.net/browse/RCT-279)
[RCT-280](https://codeandtheory.atlassian.net/browse/RCT-280)
[RCT-286](https://codeandtheory.atlassian.net/browse/RCT-286)

# Description
- Added comment in the code and video for the existing route that uses the **static route**.
- Added a link in the navigation for the redirect route demo.
- Added a dynamic route demo.
- Added default route demo.
- (RCT-279 and 280) activities are also tackled in this PR.
- (RCT-286) React Hook activity is also included in this PR to avoid merge conflict because of the multiple changes on the files.

# Validation
- Static Route
https://user-images.githubusercontent.com/93116252/229445702-aa71bd9a-f880-4d4c-8f86-2635f7ed6da0.mov

- Redirect Route
https://user-images.githubusercontent.com/93116252/229445853-f512754d-4b56-41b2-a157-747c38d9d0d4.mov

- Dynamic Route
https://user-images.githubusercontent.com/93116252/229445932-1378f104-a1e3-4ecc-9bb8-c489f3f5aa9f.mov

- Default Route
https://user-images.githubusercontent.com/93116252/229446033-c1080688-4f4b-43f0-bb04-646c82792336.mov

- RCT-286
https://user-images.githubusercontent.com/93116252/229474979-2028091e-4539-48b7-af90-ba54bd36f594.mov

# Risks
N/A

### CODE REVIEW

**Review by lycoangelo — APPROVED** [review]

File: src/components/constants/SampleText.js
Comment (lycoangelo) [inline src/components/constants/SampleText.js:1]: I think it's better to place `constants` outside components.

File: src/components/constants/SampleText.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  venenatis cursus nisi. Nulla feugiat ligula et ligula fringilla viverra.
+  In purus mauris, congue sodales tincidunt non, accumsan in est. In orci
+  turpis, scelerisque aliquet diam ac, vehicula mattis enim. Praesent
+  congue quam nunc, eu sollicitudin ipsum elementum ac. Sed semper massa
+  at purus consectetur sollicitudin. Aliquam id luctus tortor. Integer sit
+  amet elementum dolor, in pharetra diam. Donec cursus justo nec placerat
+  egestas. Sed fringilla est a diam porttitor ullamcorper. Cras vulputate
+  tincidunt sem ut vehicula.`;
+
+export default sampleText;
```
Comment (lycoangelo) [inline src/components/constants/SampleText.js:12]: No newline at EOF.

File: src/components/constants/SampleText.js
Comment (ct-aa) [reply src/components/constants/SampleText.js:1]: Thanks, @lycoangelo, This is updated now.

File: src/components/constants/SampleText.js (hunk 1 above)
Comment (ct-aa) [reply src/components/constants/SampleText.js]: Thanks, @lycoangelo. This is updated now.

**Review by lycoangelo — APPROVED** [review]

**Pull request merged**